app/invocation/invocation_timing_card.tsx

```diff
@@ -117,6 +118,16 @@ export default class InvocationTimingCardComponent extends React.Component {
       });
   }
 
+  downloadProfile() {
+    let profileFile = this.props.model.buildToolLogs?.log.find((log: any) => log.uri);
+
+    try {
+      rpcService.downloadBytestreamFile("timing_profile.gz", profileFile?.uri, this.props.model.getId());
```

Comment:
does this download the file? to your browsers download directory?

Reply:
Yep!